numba/np/random/generator_methods.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+from numba.core import types
+from numba.core.extending import overload_method
+from numba.core.imputils import Registry
+from numba.np.numpy_support import as_dtype, from_dtype
+from numba.np.random.generator_core import next_float, next_double
+
+
+registry = Registry('generator_methods')
+
+
+# Most of the standard Numpy distributions only support either
```

Comment:
This comment could go into the docstring, what do you think?

Reply:
I don't think so, this one is purely an internal functionality. The distributions in Numpy itself are the ones which support either `np.float32` or `np.float64` as `dtype` arguments. For instance: https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.standard_normal.html